canopen_master/src/emcy.cpp

```diff
@@ -42,7 +42,7 @@ struct EMCYmsg{
 
 void EMCYHandler::handleEMCY(const can::Frame & msg){
     EMCYmsg::Frame em(msg);
-    LOG("EMCY: " << can::tostring(msg, false));
+    ROSCANOPEN_ERROR("EMCY: " << msg);
```

Comment:
Please use tostring to output hex codes (much more usable):
`[ERROR] [1562077572.093793150]: 83#0000000000000000`
```suggestion
    ROSCANOPEN_ERROR("EMCY: " << can::tostring(msg, false).c_str());
```

Reply:
Can you post an example output without your fix?

Reply:
Example output without my fix (your version) is a plain decimal:
from rosout.log ```1562076301.787823008 ERROR /prbt/driver [.../emcy.cpp:45()] [topics: /rosout, /diagnostics, ...] 131

Reply:
By the way: why is the "EMCY: " string lost in both cases?

Reply:
Both issue are bugs, I am on it.

Reply:
I think I fixed it ^^

Reply:
Tried 7e45ccdce754eff61db998436cc39b7bce06c5ba and the output now reads:
```
[ERROR] [1562161991.081027610]: 83#0000000000000000
```
Note: "EMCY: " is still missing.